### DISCUSSION TITLE
debounce: any effect on performance of mesh network?

### DISCUSSION THREAD

**Discussion by bartplessers** [Koenkk/zigbee2mqtt#22658] [General]
Hi everyone,

I have a device that is quite noisy on my zigbee mesh network
https://www.zigbee2mqtt.io/devices/TS0601_air_quality_sensor.html#tuya-ts0601_air_quality_sensor

Setting a debounce value reduces the traffic - at least in the logs.
But I'm wondering: the device will still send all those messages (I assume they just don't get picked up by Z2M), so will this have any influence on the performance of my zigbee mesh network?

**Comment by TC1666** [comment]
It's a great question that I wondered also. I have one of those Tuya mmWave presence detectors and it's continuously (at least once a second) sending back distance information as I sit here in front of it and move slightly. It can't be good for the network with that flood of data. There seems to be no way to adjust the reporting frequency, I'm assuming it's just a bad/cheap firmware design by the manufacturer?

**Comment by Koenkk** [comment]
It doesnt prevent the device from sending out messages, so no change on the network performance. Z2M just limits the amount of messages send to MQTT
